src/ontology/mondo-ingest.Makefile

```diff
@@ -444,8 +444,16 @@ lexmatch/README.md: $(SCRIPTSDIR)/lexmatch-sssom-compare.py $(MAPPINGSDIR)/mondo
 
 extract-unmapped-matches: lexmatch/README.md
 
+###################################
+#### Lexmatch-combine #######
+###################################
+lexmatch/all_exact.robot.tsv: $(SCRIPTSDIR)/lexmatch-sssom-compare.py
```

Comment:
@hrshdhgd Personally I find Python way easier to do stuff in than ShellScript. But some months ago I was doing something similar for the exclusions pipeline, concatenating robot templates together but needing to leave out the 2 header lines. Stackoverflow or ChatGPT helped me create this `awk` command on 270 that worked for me.

https://github.com/monarch-initiative/mondo-ingest/blob/331e04a9ff6cba8aa7d41b439d011124ac54796d/src/ontology/mondo-ingest.Makefile#L268-L270

I don't know if this will work for you though because I see that you're also doing `merged_lex_exact_df.drop_duplicates(inplace=True)`. I don't know how I would do that in ShellScript.

Reply:
Yeah since the python script file was already in the project I overloaded it with another CLI command. Makes it easier to maintain and debug. But thanks for this, I always wondered what `awk` did.